routes/error-tracker.js

```diff
@@ -49,6 +51,27 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} stackTrace
+ * @return {string} converted stackTrace
+ */
+function convertStackTrace(stackTrace) {
+  let validStackTrace = '';
+  if (chromeStackTraceRegex.exec(stackTrace)) {
+    return stackTrace.match(chromeStackTraceRegex).join('\n');
```

Comment:
This looks like it's doing duplicate work by running both `exec()` and `match()` for the same string. Also, do we even need to join the matched result? Why not just return the original stack trace?

```js
if (chromeStackTraceRegex.test(stackTrace)) {
  return stackTrace;
}
```